scrapy/utils/python.py

```diff
@@ -194,7 +194,8 @@ def _getargspec_py23(func):
 def get_func_args(func, stripself=False):
     """Return the argument name list of a callable"""
     if inspect.isfunction(func):
-        func_args, _, _, _ = _getargspec_py23(func)
+        spec = inspect.getfullargspec(func)
+        func_args = spec.args + spec.kwonlyargs
```

Comment:
Is it a cleanup or an issue fix? If that's an issue fix, could you please add a test case for get_func_args, even if without the fix a different part of scrapy fails?